### PR TITLE
List all organizations

### LINKED ISSUE #1323: Listing of all organizations
**Is your feature request related to a problem? Please describe.**
Add a screen to monitor organizations status (Active/Inactive/Awaiting approval)

**Describe the solution you'd like**
Listing of all organizations split into 3 groups/status
* Active Organizations
* Organization awaiting approval
* Organization in-active
* For active organizations, we should have a Make Inactive button
* For in-active organizations, we should have a Make Active, and Delete Organization button. The delete organization should have a confirmation and make the user type the full name of the organization
* For organizations awaiting approval, we should add an Approve and Delete action, which marks the organization as approved and/or deletes the organization

### PR DESCRIPTION
**Pull request by shreesh1993**
- Added organization list
- Now user can perform actions such as activate, approve and delete.
- There will be a confirmation message while deleting in-active/invalid organization.

closes  #1323

### CODE REVIEW

File: src/graphql/queries/Organization.ts [hunk 1]
```diff
@@ -90,3 +90,33 @@ export const BSPBALANCE = gql`
...
   }
 `;
+
+export const GET_ORGANIZATION_COUNT = gql`
+  query countOrganizations($filter: OrganizationFilter!) {
+    countOrganizations(filter: $filter)
+  }
+`;
+
+export const FILTER_ORGANIZATIONS = gql`
```
Comment (DigneshGujarathi) [inline src/graphql/queries/Organization.ts:100]: both are the same - FILTER_ORGANIZATIONS, FILTER_ORGANIZATIONS_NAME, please remove/reuse

File: src/assets/images/icons/Organisation-1.svg [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (DigneshGujarathi) [inline src/assets/images/icons/Organisation-1.svg:1]: Are we using this?

File: src/assets/images/icons/Organisation-2.svg [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (DigneshGujarathi) [inline src/assets/images/icons/Organisation-2.svg:1]: are we using this?

**Review by DigneshGujarathi — CHANGES_REQUESTED** [review]

File: src/config/menu.ts [hunk 4]
```diff
@@ -55,6 +55,12 @@ const menus = [
     type: 'staffManagement',
     roles: ['Staff', 'Manager', 'Admin'],
   },
+  {
+    title: 'Organization',
+    path: '/organization',
+    type: 'staffManagement',
+    roles: ['Manager', 'Admin'],
```
Comment (mdshamoon) [inline src/config/menu.ts:62]: Just give rights for admin only.

**Review by mdshamoon — COMMENTED** [review]
Please change organization to organizations everywhere.

File: src/context/role.ts [hunk 5]
```diff
@@ -40,6 +42,15 @@ export const setUserRolePermissions = () => {
     sideDrawerMenu = getMenus('sideDrawer', 'Manager');
     staffManagementMenu = getMenus('staffManagement', 'Manager');
 
+    /**
+     * Removing organization list menu for non admin users
+     */
+    if (user.name !== 'NGO Main Account') {
```
Comment (mdshamoon) [inline src/context/role.ts:48]: The user will be Saas Admin. Let's discuss and confirm in the call.

**Review by mdshamoon — APPROVED** [review]

**Pull request merged**